src/csharp/Microsoft.Spark.Experimental/Sql/RelationalGroupedDatasetExtensions.cs

```diff
@@ -33,6 +34,14 @@ public static class RelationalGroupedDatasetExtensions
         /// </param>
         /// <param name="func">A grouped map user-defined function.</param>
         /// <returns>New DataFrame object with the UDF applied.</returns>
+        public static DataFrame Apply(
+            this RelationalGroupedDataset dataset,
+            StructType returnType, 
+            Func<FxDataFrame, FxDataFrame> func)
+        {
+            return dataset.Apply(returnType, func);
+        }
+
         public static DataFrame Apply(
```

Comment:
Should we have xml docs comments for this method?

Reply:
Yup. Fixed.